sdk/search/azure-search/README.md

```diff
@@ -0,0 +1,252 @@
+# Azure cognitive search client library for Java
+
+This is the Java client library for [Azure Cognitive Search](https://docs.microsoft.com/en-us/rest/api/searchservice/).
+Azure Cognitive Search is a fully managed cloud search service that provides a rich search experience to custom applications.
+This library provides an easy (native) way for a Java developer to interact with the service to accomplish tasks like: create and manage indexes, load data, implement search features, execute queries, and handle results.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]
+| [Product documentation][search_docs] | [Samples][samples]
+
+## Getting started
+
+### Prerequisites
+
+-   Java Development Kit (JDK) with version 8 or above
+-   [Azure subscription][azure_subscription]
+-   An instance of [Cognitive search service][search]
+
+### Adding the package to your product
+
+[//]: # "{x-version-update-start;com.azure:azure-search;current}"
```

Comment:
The quotes aren't necessary and the begin/end tags should be:
[//]: # ({x-version-update-start;com.azure:azure-search;current})
[//]: # ({x-version-update-end})

The reason for the specific tag syntax is that these update tags, the way they are don't show up when the user looks views the md file and they get omitted when the readme.md is committed to html and added to the doc's landing page.